dspace-spring-rest/src/main/java/org/dspace/app/rest/model/SearchFacetEntryRest.java

```diff
@@ -87,4 +96,28 @@ public int getFacetLimit() {
     public void setFacetLimit(final int facetLimit) {
         this.facetLimit = facetLimit;
     }
+
+    public boolean isExposeMinMax() {
```

Comment:
Methods in this class need javadocs

Reply:
Wouldn't it be better to rename the method to `exposesMinAndMaxValue` and to not add documentation. I'm not sure documentation will have a much added value here.

Reply:
Again, JavaDocs for all public methods are in our Code Style (see comment above).  Renaming the method is also more than welcome (it is awkwardly named), but JavaDocs are still requested.